### ISSUE TITLE
Samples don't work for VS 2019

### ISSUE THREAD

**Issue by mtanneryd** [telerik/telerik-xamarin-forms-samples#12]

**Comment by didiyordanova** [comment]
Hi @mtanneryd ,

Could you please elaborate more on the reported issue? How the "Samples don't work for VS 2019"?

**Further information**
May I ask you to send us a stack trace of the exception you get? Also of a great help will be the following information:

- versions of Xamarin.Forms and Telerik UI for Xamarin 
- version of Windows and exact version of VS 2019
- where the issue could be reproduced: running the samples on iOS, Android or UWP

Could you please specify which sample doesn't work: **The Tagit Demo app** or **the Samples Application (QSF Demo App)**?

Thank you in advance and I am looking forward to your reply.

**Comment by mtanneryd** [comment]
I'm using the latest versions of Xamarin.Forms, Telerik UI for Xamarin and VS 2019. Downloading the sample solution and opening it in VS 2019 it does not compile. Adding the Telerik NuGet server does not help.

The sample solution comes with references to the trial telerik packages but I have a licensed version. Unfortunatly following the instructions to uninstall the trial packages and reinstalling the licensed packages does not work since uninstalling fails saying that it is unable to find the 2019.2.513.1 version. It claims that it is "not available in this source".

VS 2019: 16.2.0
Telerik UI for Xamarin: 2019.2.705.1
Xamarin.Forms: 3.6.0.264807

**Comment by didiyordanova** [comment]
Hi @mtanneryd ,

Thank you for the provided information.

I was able to investigate it further and indeed the Trial nugets are not visible if you have only purchased licence. In my case I have both purchased and trial so that is why I was able to find them on the nuget server.

I strongly recommend just **manually editing the csproj files** in all projects, directly change the version of the Telerik UI for Xamarin package and reload the solution. 
For example, inside the tagit.csproj file
replace
`<PackageReference Include="Telerik.UI.for.Xamarin.Trial" Version="2019.2.708.1" />`

with 
`<PackageReference Include="Telerik.UI.for.Xamarin" Version="2019.2.708.1" />`

Give this a try and let me know how it goes.
